### PR TITLE
fix(roc-package-web-app-react): Add ApolloClient to dev Provider

### PR DESCRIPTION
**Pull request by glennlegend**
Adds a client prop to the dev provider if HAS_APOLLO is set.

### CODE REVIEW

**Comment by dlmr** [comment]
Thanks for the PR!

What does this do when using Apollo, does not the Redux Devtools work correctly without it?

**Comment by glennlegend** [comment]
AFAIK if HAS_APOLLO, the `Provider` will be a ApolloProvider so it will fail because it needs the client prop.

**Comment by glennlegend** [comment]
image

File: packages/roc-package-web-app-react/app/client/create-client.js [hunk 1]
```diff
@@ -166,8 +166,14 @@ export default function createClient({
...
 
+                    const devProps = { store };
+
+                    if (HAS_APOLLO) {
+                        devProps.client = providerProps.client;
+                    }
+
                     createDevComponent.push((component) => (
-                        <Provider store={store}>
+                        <Provider {...devProps}>
```
Comment (dlmr) [inline packages/roc-package-web-app-react/app/client/create-client.js]: Let's use `providerProps` directly here.

**Pull request merged**